_data/simple-icons.json

```
@@ -3533,6 +3533,11 @@
             "hex": "88CE02",
             "source": "https://greensock.com/"
         },
+        {
+            "title": "Grid AI",
+            "hex": "78FF96",
+            "source": "https://github.com/gridai/logos/blob/1e12c83b77abdc22a41566cab232f4db40223895/GridAI-icons/icon-white-48.svg"
```

In the interests of getting this merged for this weekend's release, let's run with "Grid.ai" if we don't hear back from them by the end of today. Like I said, if they would prefer a different stylisation, we can do that in next week's release or, if it constitutes a breaking change, the following week which will be our next major release.

_If_ they do request a different stylisation then we'll need to add "Grid.ai" as an alias for people who will be looking for it using that name.
```suggestion
            "source": "https://github.com/gridai/logos/blob/1e12c83b77abdc22a41566cab232f4db40223895/GridAI-icons/icon-white-48.svg",
            "aliases": {
                "aka": [
                    "Grid.ai"
                ]            
```

---

the alias option  `grid.ai` for original `grid ai` looks as the best for to me